Add Wan2.2-I2V-A14B support

AChowdhury1211 opened the pull request:
- Added support for Wan2.2_I2V_A14B 
- Utilized the I2V_generate and T2V_generate methods.
- Added thresh_i2v in arguments

AChowdhury1211 committed:
Added support for Wan2.2_I2V_A14B and utilized the I2V_generate and T2V_generate methods.

LMD0311 commented:
I have also been ready for the I2V task, but further testing is needed. I'll merge this as soon as possible.

LMD0311 committed:
Add EasyCache support for Wan2.2 I2V-A14B

Introduces EasyCache acceleration for Wan2.2 I2V-A14B, including new example assets and updated documentation with usage instructions and visual comparisons. Refactors easycache_generate.py to remove duplicated generation functions and streamline global variable setup for T2V and I2V tasks. Updates README files to reflect new model support and release information.

LMD0311 reviewed (approved):
Ready for merge.
